src/arm/linux/aarch64-isa.c

```diff
@@ -151,4 +151,25 @@ void cpuinfo_arm64_linux_decode_isa_from_proc_cpuinfo(
 	if (features & CPUINFO_ARM_LINUX_FEATURE_ASIMDFHM) {
 		isa->fhm = true;
 	}
+
+#if defined(__linux__)
```

Comment:
Why does it needs `#ifdef __linux__`, as this file should only be compiled for Linux anyway, isn't it?

Reply:
Agree. Removed this macro.